docs/en/rules/Azure.Databricks.SKU.md

```diff
@@ -0,0 +1,73 @@
+---
+reviewed: 2024-01-12
+severity: Critical
+pillar: Performance Efficiency
+category: Scaling and Partitioning
```

Comment:
[PE:03 Selecting services](https://learn.microsoft.com/azure/well-architected/performance-efficiency/select-services) might be a better fit for this one. But if you disagree, let me know.

```suggestion
category: PE:03 Selecting services
```